### PR TITLE
trim the gpu image names for proper image-pull

### PR DESCRIPTION
**Pull request by harshad16**
trim the gpu image names for proper image-pull
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

File: jupyterhub/notebook-images/overlays/cuda-11.0.3/gpu-notebook.yaml [hunk 1]
```diff
@@ -42,7 +42,7 @@ items:
     output:
       to:
         kind: ImageStreamTag
-        name: 's2i-minimal-gpu-cuda-$(cuda_version)-notebook:latest'
+        name: 'minimal-gpu:cuda-$(cuda_version)'
```
Comment (harshad16) [inline jupyterhub/notebook-images/overlays/cuda-11.0.3/gpu-notebook.yaml:45]:
```
        name: 'minimal-gpu:cuda-$(cuda_version)-py3.8'
        or
        name: 'minimal-py3.8-gpu:cuda-$(cuda_version)'
```
should we define python version as well.

Comment (tmckayus) [reply jupyterhub/notebook-images/overlays/cuda-11.0.3/gpu-notebook.yaml:45]: @harshad16 I think the first one is good. Do we do the same thing for tf and pytorch?

**Review by crobby — APPROVED** [review]

File: jupyterhub/notebook-images/overlays/cuda-11.0.3/gpu-notebook.yaml (hunk 1 above)
Comment (harshad16) [reply jupyterhub/notebook-images/overlays/cuda-11.0.3/gpu-notebook.yaml:45]: we can same thing for them too

**Comment by harshad16** [comment]
Ready for review again.

**Review by crobby — APPROVED** [review]

**Pull request merged**